backend/composer/services/graph_service.py

```diff
@@ -4,6 +4,13 @@
 JOURNEY_DELIMITER = '\\'
 
 
+def get_entity_name(entity):
```

Comment:
suggestion: This is most likely introduced before https://github.com/MetaCell/sckan-composer/pull/257/commits/01bba38fc65ee6017c14b0dd6aa1ebbb9b46d61c got merged into develop. With the name being a @property of AnatomicalEntity I don't expect us to need this `get_entity_name` anymore

Reply:
@afonsobspinto I agree

Reply:
I don't agree, the above makes the naming we use in the dropdown consistent with the journey and makes the journey/preview more readable.

Current preview:
```
A connection goes from descending colon,Myenteric nerve plexus of descending colon to inferior mesenteric ganglion via descending colon,Longitudinal muscle layer of descending colon via descending colon,Serosa of descending colon via lumbar colonic nerve.
```

with get_entity_name:
```
A connection goes from descending colon (Myenteric nerve plexus of descending colon) to inferior mesenteric ganglion via descending colon (Longitudinal muscle layer of descending colon) via descending colon (Serosa of descending colon) via lumbar colonic nerve.
```

Reply:
@afonsobspinto I introduced that method yesterday since the current name for AE is region,layer and it makes difficult read/interpret the journey since that is the same separator we use for multiple entities in a layer, I get lost of what is a simple entity, what a region and what a layer.
With that method I am just respecting the convention we put in the acceptance criteria to express region (layer) through the UI, but since this is generated backend wise that is why I added such method.

Reply:
@ddelpiano but shouldn't we always use the same convention? Is so then i suggest to fix/change the commit if @afonsobspinto

Reply:
@zsinnema fine by me

Reply:
I just changed the name property to match with @ddelpiano request:

![image](https://github.com/MetaCell/sckan-composer/assets/19196034/d7a92ecd-4902-49eb-bb31-907af2b30ae7)

Reply:
lgtm